packages/template-ui/src/App.vue

```diff
@@ -58,9 +58,14 @@ export default {
       this.$store.commit('setHomeNavigation');
       const uri = window.location.search.substring(1);
       const params = new URLSearchParams(uri);
+      // Check if we need to navigate to a specific content or topic. Content takes precedence.
+      const contentId = params.get('contentId');
+      if (contentId) {
+        this.$router.push(`/c/${contentId}`);
+      }
       const topicId = params.get('topicId');
       if (topicId) {
-        this.$router.push(`/${topicId}`);
+        this.$router.push(`/t/${topicId}`);
```

Comment:
It looks like this is going to behave differently from the comment (and some poking with Firefox's dev tools seems to confirm it). If there's both a `contentId` and a `topicId`, we're going to call `this.$router.push` _twice_. Perhaps this would be better as an `if / else if`?

Reply:
I don't know if the case of passing the `topicId` and the `contentId` is something that could happen, but I don't think so.

Reply:
It should never happen, but it's a very good point. I will make it an if/else if.

Reply:
Done, I prefer early return than nested conditionals.